Test/Optimizer/test_sr_itersolve.py

```diff
@@ -65,8 +65,6 @@ def test_qgt_solve(qgt, vstate, solver, _mpi_size, _mpi_rank):
     S = qgt(vstate)
     x, _ = S.solve(solver, vstate.parameters)
 
-    jax.tree_multimap(lambda a, b: np.testing.assert_allclose(a, b), S @ x, vstate.parameters)
```

Comment:
@PhilipVinc is this incorrect? I think we should check whether `x` actually solves S.x=y!

Reply:
Also, there are so many reasons it might not work...
I mean, you don't necessarily converge to the right solution. especially with a neural network with random weights.